docs/se/guides/07_db_client.adoc

```diff
@@ -0,0 +1,634 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2021 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Helidon SE DB Client Guide
+:h1Prefix: SE
+:description: Helidon db-client
+:keywords: helidon, db, client
+
+This guide describes Helidon DB Client features and how to create a sample Helidon SE project
+that can be used to run some basic example using helidon db client.
+
+== What you need
+
+[width=50%,role="flex, sm7"]
+|===
+|About 15 minutes
```

Comment:
For this 15 minute tutorial, you will need the following:
include::{common-page-prefix-inc}[tag=common-prereqs]